distribution/lib/Standard/Base/0.0.0-dev/src/Data/Json.enso

```diff
@@ -210,7 +210,8 @@ type JS_Object
     to_default_visualization_data self =
         render self
 
-    ## Creates an Enso object from the JS_Object.
+    ## GROUP Conversions
```

Comment:
Why do we not prefix this as `Standard.Base.Conversions`?

Reply:
Is it a scoping thing? It's implied since this is in `Standard.Base`?

Reply:
Yes it's implied as in Standard.Base.